airflow/opt/airflow.cfg

```diff
@@ -94,15 +94,15 @@ load_examples = False
 load_default_connections = False
 
 # Path to the folder containing Airflow plugins
-plugins_folder = /opt/airflow/plugins
+plugins_folder = /opt/airflow/providers/etna/etna/plugins
```

Comment:
I needed to update this path in order for the plugins to be found...

Reply:
That makes total sense.  I think I started out with a general plugins dir but deprecated it, but now we need it.  The plugins / providers difference is a bit confusing tbh.